services/QuillLMS/app/queries/activity_feedback_history.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ActivityFeedbackHistory
+  def self.run(activity_id:)
+    activity_stats_query(activity_id: activity_id)
+  end
+
+  def self.activity_stats_query(activity_id:)
+    activity_sessions_agg = RawSqlRunner.execute(
+      <<-SQL
+        SELECT
```

Comment:
I haven't tested this, but it might be more performant to do the outer query in AREL, so that the current subquery is elevated to a vanilla, non sub-query.

Reply:
I thought about this, but I wasn't quite sure how to use AREL with a query that's not selecting from a table. So instead of `Users.select(.....)` I wasn't sure what to put on the left side of that `.select`, since I'm selecting from a subquery and not a specific table?